#! /usr/bin/env python

PACKAGE='youbot_scanner_lines'
import roslib; roslib.load_manifest(PACKAGE)

from dynamic_reconfigure.parameter_generator import *


gen = ParameterGenerator()
#       Name                    					Type      Reconfiguration level             Description                                                 	Default    Min   Max
gen.add("scanner_lines_filter_dequeSize",   	    int_t,  0, 							    	"How many msgs should be processed for the average",           	10,    		1, 	  100)

exit(gen.generate(PACKAGE, "dynamic_reconfigure_node", "ScannerLinesFilter"))
